src/main/java/org/interledger/InterledgerAddress.java

```diff
@@ -29,7 +30,27 @@
 public interface InterledgerAddress {
 
   /**
-   * Return this address's value as a non-null {@link String}.  For example:
+   * Constructor to allow quick create of String.
+   *
+   * @param value String representation of an Interledger Address
+   */
+  static InterledgerAddress of(final String value) {
```

Comment:
FYI - the `public` modifier is redundant on an interface because it's implied.

For years while I was an eclipse user, I would add `public` to all interface methods (and other redundant modifiers like `public static final` on a String in an interface). Then, when I switched to IntelliJ, the default setting is to grey-out redundant modifiers, and now I've come to love that style.

Yet another reason you'll come to love IntelliJ!!